hadoop-hdfs-project/hadoop-hdfs-rbf/src/main/java/org/apache/hadoop/hdfs/server/federation/router/RouterRpcClient.java

```diff
@@ -484,14 +486,14 @@ private RetryDecision shouldRetry(final IOException ioe, final int retryCount,
    * Invokes a method against the ClientProtocol proxy server. If a standby
    * exception is generated by the call to the client, retries using the
    * alternate server.
-   *
```

Comment:
Does this give warnings? If no, it is easier to read with the break line.
The other option is to explicitly add the breakline.

Reply:
done, added breakline